docs: offline enterprise license (#4327)
apollographql/router: docs/source/configuration/overview.mdx

The [graph API key](/graphos/api-keys/#graph-api-keys) that the router should use to authenticate with GraphOS when fetching its supergraph schema.

> If starting the router with an [offline license](#--license), setting both `APOLLO_GRAPH_REF` and `APOLLO_KEY` configures the router to send usage metrics to GraphOS on a best-effort basis; not setting both disables sending usage metrics. Otherwise, setting this is **required** when using managed federation.

Reviewer: Since this applies to both and is repeated, it might make more sense to put it in a subsection below the table. Then, in each row in the table it can say something like this:

```suggestion
**Required** when using [managed federation](/federation/managed-federation/overview/) except when using an [offline license](#offline-license) to run the router.
```

##### `-s` / `--supergraph`

> `APOLLO_ROUTER_SUPERGRAPH_PATH`, `APOLLO_ROUTER_SUPERGRAPH_URLS`

Reviewer 2: Let's also explain the difference between these and indicate that URLs are comma separated.
These URLS will potentially end up in log messages, so users should take care not to embed tokens in the URL.

<tr>
<td style="min-width: 150px;">

##### `--license`

> `APOLLO_ROUTER_LICENSE_PATH`, `APOLLO_ROUTER_LICENSE`

Reviewer 2: Is it worth explaining the difference between these?

</td>
<td>

> An offline GraphOS Enterprise license. Enables Enterprise router features when disconnected from GraphOS. 

Reviewer: Since it accepts a path via the `APOLLO_ROUTER_LICENSE_PATH` option, maybe include "The absolute or relative path to the Apollo Router's license"?
